modules/xfem/include/efa/EFAElement3D.h

```diff
@@ -105,8 +105,7 @@ class EFAElement3D : public EFAElement
   unsigned int getFaceID(EFAFace * face) const;
   std::vector<unsigned int> getCommonFaceID(const EFAElement3D * other_elem) const;
   bool getCommonEdgeID(const EFAElement3D * other_elem,
-                       std::vector<unsigned int> & face_id,
-                       std::vector<unsigned int> & edge_id) const;
+                       std::vector<std::pair<unsigned int, unsigned int>> & common_ids) const;
```

Comment:
A lot of this code predates our use of doxygen comments, but we have been slowly adding them as we make changes. It would be especially helpful to add a doxygen comment to this method to clarify the fact that it populates `common_ids`.